tests/driver/napi/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+# Copyright © SixtyFPS GmbH <info@slint.dev>
+# SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-Royalty-free-1.1 OR LicenseRef-Slint-commercial
+
+[package]
+name = "test-driver-napi"
+authors.workspace = true
+edition.workspace = true
+homepage.workspace = true
+license.workspace = true
+repository.workspace = true
+rust-version.workspace = true
+version.workspace = true
+publish = false
+
+[[bin]]
+path = "main.rs"
+name = "test-driver-napi"
+
+[dev-dependencies]
+test_driver_lib = { path = "../driverlib" }
+# Require `artifact`  dependencies tracked by rust RFC 3028
+#slint-node = { path = "../../../api/napi", artifact = ["cdylib"] }
```

Comment:
It looks like that despite this being in a comment for future adaptation, the path and name should be changed.